server/lib/nl/config_builder/timeline.py

```diff
@@ -78,8 +78,7 @@ def ranked_timeline_collection_block(builder: base.Builder,
       else:
         chart_title = base.decorate_chart_title(title=sv2thing.name[sv_dcid],
                                                 place=place)
-      facet_id = get_facet_id(sv_dcid, single_date, cspec.sv_place_facet_id,
```

Comment:
Why is this removed?

Reply:
it isn't removed, the function is just updated to not take date as a param anymore. `get_facet_id` used to take a `date` as a param and only return an id if there is a date passed in. We now want to get facet id even in non date situations, so I updated the function to get facet id if sv_place_facet_id is not empty.

Reply:
Got it.